src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -519,6 +522,17 @@ export const WorkflowView = _.flow(
                     sourceRepo,
                     methodPath: sourceRepo === 'agora' ? `${methodNamespace}/${methodName}` : methodPath,
                   });
+                  // TODO should this be done here or when the workflows are actually launched?
+                  // What info needs to be included?
+                  if (perWorkflowCostCap !== '') {
```

Comment:
The "workflow launch" metric is logged here, when the "launch analysis modal" is opened but before the workflow is actually launched.  I put the "setWorkflowCostCap" metric here along with it, but it could be a false positive if the user cancels the launch.  Should I leave it here?  Move them both?

Reply:
It seems useful to have the metric recorded here so we can see if users are setting the cost cap but ultimately canceling their workflow before launch for whatever reason. We can still see how many people are actually launching capped workflows in Rawls, but maybe this would tell us if the UI could be easier to understand?